### DISCUSSION TITLE
Fix position

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7291] [Help] [answered]
How can we fix any component position while scrolling

**Comment by andz-bb** [comment, marked as answer]
Hey @DevrayePatil - you can add custom CSS to Budibase components to fix their position.

As an example, I created a button and pressed Edit custom CSS to add my own styles.
![Edit custom styles](https://user-images.githubusercontent.com/110921612/184848322-249f071d-8c8f-4b3f-a4d4-2469183c4619.png)

I then added CSS that made the component's position fixed and defined where I wanted it to be fixed to.

![fixed position css](https://user-images.githubusercontent.com/110921612/184848711-1ee43496-590c-4eaf-a423-876f79dc4396.png)

Now the button is fixed at that position in the page as I scroll up and down it.
![scrolling with fixed button](https://user-images.githubusercontent.com/110921612/184849496-50def1bf-8995-485c-ab7d-c3357fe4fe8f.gif)


Something like this could work for you or you can find more detailed tutorials online for doing this with CSS. Here is the code as a really basic example:

```
position: fixed; 
top: 80%; 
right: 16%;
```

Hope this helps! If you have any further issues with it let me know and I'll be happy to look into it further ☺️
